src/imxrt1060/pads.rs

```diff
@@ -0,0 +1,2211 @@
+    // GPIO_SPI_B1_05 does not have a GPIO alternate.
+
+    const GPIO_SPI_B1_06_MUX_ADDR: u32 = 0x401f86ac;
+    const GPIO_SPI_B1_06_PAD_ADDR: u32 = 0x401f8704;
+    pub type GPIO_SPI_B1_06 = crate::Pad<GPIO_SPI_B1_06_MUX_ADDR, GPIO_SPI_B1_06_PAD_ADDR>;
+    // GPIO_SPI_B1_06 does not have a GPIO alternate.
+
+    const GPIO_SPI_B1_07_MUX_ADDR: u32 = 0x401f86b0;
+    const GPIO_SPI_B1_07_PAD_ADDR: u32 = 0x401f8708;
+    pub type GPIO_SPI_B1_07 = crate::Pad<GPIO_SPI_B1_07_MUX_ADDR, GPIO_SPI_B1_07_PAD_ADDR>;
+    // GPIO_SPI_B1_07 does not have a GPIO alternate.
+
+    /// All pads with prefix GPIO_SPI_B1.
+    pub struct Pads {
+        pub p00: GPIO_SPI_B1_00,
+        pub p01: GPIO_SPI_B1_01,
+        pub p02: GPIO_SPI_B1_02,
+        pub p03: GPIO_SPI_B1_03,
+        pub p04: GPIO_SPI_B1_04,
+        pub p05: GPIO_SPI_B1_05,
+        pub p06: GPIO_SPI_B1_06,
+        pub p07: GPIO_SPI_B1_07,
+    }
+    /// Erased pads with prefix GPIO_SPI_B1.
+    ///
+    /// Use [`Pads::erase()`] to get an `ErasedPads` instance.
+    pub type ErasedPads = [crate::ErasedPad; 8];
+    impl Pads {
+        /// Take all pads from this group
+        ///
+        /// # Safety
+        ///
+        /// You may safely call this once to acquire all of the pads.
+        /// Subsequent calls may return pads that are mutably aliased
+        /// elsewhere. Consider calling new() at the start of your program.
+        #[inline]
+        pub const unsafe fn new() -> Self {
+            Self {
+                p00: GPIO_SPI_B1_00::new(),
+                p01: GPIO_SPI_B1_01::new(),
+                p02: GPIO_SPI_B1_02::new(),
+                p03: GPIO_SPI_B1_03::new(),
+                p04: GPIO_SPI_B1_04::new(),
+                p05: GPIO_SPI_B1_05::new(),
+                p06: GPIO_SPI_B1_06::new(),
+                p07: GPIO_SPI_B1_07::new(),
+            }
+        }
+
+        /// Erase all of the pads
+        ///
+        /// The return type is an array, where the index indicates the
+        /// pad offset from the start of the group. For example, AD_B0_03
+        /// would be referenced as erased_pads[3].
+        ///
+        /// See `ErasedPads` for more information.
+        #[inline]
+        pub const fn erase(self) -> ErasedPads {
+            [
+                self.p00.erase(),
+                self.p01.erase(),
+                self.p02.erase(),
+                self.p03.erase(),
+                self.p04.erase(),
+                self.p05.erase(),
+                self.p06.erase(),
+                self.p07.erase(),
+            ]
+        }
+    }
+}
+
+/// All of the pads.
+pub struct Pads {
+    pub gpio_emc: gpio_emc::Pads,
+    pub gpio_ad_b0: gpio_ad_b0::Pads,
+    pub gpio_ad_b1: gpio_ad_b1::Pads,
+    pub gpio_b0: gpio_b0::Pads,
+    pub gpio_b1: gpio_b1::Pads,
+    pub gpio_sd_b0: gpio_sd_b0::Pads,
+    pub gpio_sd_b1: gpio_sd_b1::Pads,
+    pub gpio_spi_b0: gpio_spi_b0::Pads,
+    pub gpio_spi_b1: gpio_spi_b1::Pads,
```

Comment:
Note that the 1060 has two new pad groups. I've not found direct mention of these pads in the 1060 reference manual, but they seem to be in the SVDs.